test/index.spec.js

```diff
@@ -154,8 +193,9 @@ function runTestWithConfig (testConfig, initialStoreState, label) {
       })
 
       it('should increment when action is dispatched to store', () => {
+        let expectedResult = store.getState().present + 1
         store.dispatch({type: 'INCREMENT'})
-        expect(store.getState()).to.deep.equal(incrementedState)
+        expect(store.getState().present).to.equal(expectedResult)
```

Comment:
We can't rely on `incrementedState` here, We're going through the store and history might have changed by other tests. This is kindof a bonus test so it's fine here